### PR TITLE
[RW-2596][risk=no] stigmatization table

### PR DESCRIPTION
**Pull request by jacmrob**
Screen Shot 2019-05-30 at 12 45 55 PM

resizes appropriately

### CODE REVIEW

File: ui/src/app/views/workspace-edit.tsx [hunk 1]
```diff
@@ -722,8 +773,27 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
...
+            More info on stigmatization
+            {this.state.showStigmatizationDetails ? <ClrIcon shape='caret' dir='up'/> :
+              <ClrIcon shape='caret' dir='down'/>}
+          </Link>
+          {this.state.showStigmatizationDetails &&
+            <div>
+              <div style={styles.text}>
+                Populations that are historically medically underserved or underrepresented in
+                biomedical research are also more vulnerable to stigmatization. If your population
+                of interest includes the following categories defined as UBR by the All of Us
```
Comment (calbach) [inline ui/src/app/views/workspace-edit.tsx]: text: UBR should be spelled out here - UBR in parens if desired

File: ui/src/app/views/workspace-edit.tsx [hunk 2]
```diff
@@ -722,8 +773,27 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
...
+                Research Program, you are encouraged to request a review of your research
+                purpose by the RAB.
```
Comment (calbach) [inline ui/src/app/views/workspace-edit.tsx]: text: RAB should be spelled out here - RAB in parens if desired

File: ui/src/app/views/workspace-edit.tsx [hunk 3]
```diff
@@ -722,8 +773,27 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
             </div>
           </div>
         </WorkspaceEditSection>
-        <WorkspaceEditSection header='Request a review of your research purpose'
+        <WorkspaceEditSection header='Request a review of your research purpose for potential
+                                      stigmatization of Research Participants'
```
Comment (calbach) [inline ui/src/app/views/workspace-edit.tsx]: text: I'm confused by capitalization of "Research Participants". It's not used anywhere else on the page and I don't think this is anything like a proper phrase or legal term we've defined elsewhere.

File: ui/src/app/views/workspace-edit.tsx [hunk 4]
```diff
@@ -722,8 +773,27 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
             </div>
           </div>
         </WorkspaceEditSection>
-        <WorkspaceEditSection header='Request a review of your research purpose'
+        <WorkspaceEditSection header='Request a review of your research purpose for potential
```
Comment (calbach) [inline ui/src/app/views/workspace-edit.tsx]: Note: the mocks have Research Purpose capitalized. I don't agree with that capitalization though (I'd keep it how you have it here)..

File: ui/src/app/views/workspace-edit.tsx [hunk 5]
```diff
@@ -302,6 +329,28 @@ export const LabeledCheckBox = (props) => {
   </div>;
 };
 
+export const UbrRow = (props) => {
```
Comment (calbach) [inline ui/src/app/views/workspace-edit.tsx]: nit: this is really more a table cell than a row

File: ui/src/app/views/workspace-edit.tsx [hunk 6]
```diff
@@ -130,34 +130,57 @@ export const researchPurposeQuestions = [
...
-    object: SpecificPopulationEnum.DISABILITYSTATUS
+    object: SpecificPopulationEnum.DISABILITYSTATUS,
+    ubrLabel: 'Disability Status',
+    ubrDescription: 'Physical and Cognitive Disabilities'
   }, {
     label: 'Access to care',
-    object: SpecificPopulationEnum.ACCESSTOCARE
+    object: SpecificPopulationEnum.ACCESSTOCARE,
+    ubrLabel: 'Access to Care',
+    ubrDescription: 'Limited Access to care; Cannot easily obtain or access medical care'
```
Comment (calbach) [inline ui/src/app/views/workspace-edit.tsx]: nit: lowercase access

File: ui/src/app/views/workspace-edit.tsx [hunk 7]
```diff
@@ -130,34 +130,57 @@ export const researchPurposeQuestions = [
 export const specificPopulations = [
   {
     label: 'Race/Ethnicity',
-    object: SpecificPopulationEnum.RACEETHNICITY
+    object: SpecificPopulationEnum.RACEETHNICITY,
+    ubrLabel: 'Ancestry (Race/Ethnicity)',
+    ubrDescription: 'American Indian and Alaska Native (AIAN); Black, African American, or \
+       African; Middle Eastern or North African (MENA); Native Hawaiian or Other Pacific \
```
Comment (calbach) [inline ui/src/app/views/workspace-edit.tsx]: nit: I would probably use + continuation, `\` should be injecting the indented spaces as part of the string. It probably renders normally as HTML, but feels a little wrong to depend on that.

File: ui/src/app/views/workspace-edit.tsx (hunk 1 above)
Comment (jacmrob) [reply ui/src/app/views/workspace-edit.tsx]: karthik approved this; fixing

File: ui/src/app/views/workspace-edit.tsx (hunk 2 above)
Comment (jacmrob) [reply ui/src/app/views/workspace-edit.tsx]: karthik approved this; fixing

File: ui/src/app/views/workspace-edit.tsx (hunk 3 above)
Comment (jacmrob) [reply ui/src/app/views/workspace-edit.tsx]: karthik approved, changing to lowercase

File: ui/src/app/views/workspace-edit.tsx (hunk 4 above)
Comment (jacmrob) [reply ui/src/app/views/workspace-edit.tsx]: karthik is fine with it lowercase

**Pull request merged**